src/install.ts

```diff
@@ -23,6 +23,18 @@ export default function install(): void {
 		osxKeyBinding: 'cmd+alt+m',
 	});
 
+	// This is a test annotation type that is only enabled for a specific document.
+	registerTextRangeReviewAnnotationType('enabled-selector-comment', {
+		enabledSelector: 'fonto:document(fonto:remote-document-id(fonto:selection-common-ancestor()))/*[1]/@id = "topic_915c9f1b-ee2a-4853-d8f2-5fffaa9ab116"',
```

Comment:
Can you change this to compare on the remote-document-id instead? That's more readable and does the same :)